jobs/scripts/nightly-samba-builds/nightly-samba-builds.sh

```diff
@@ -33,13 +33,15 @@ yum -y install git make rpm-build mock ansible createrepo_c
 
 yum -y install \
 	qemu-kvm \
-	qemu-kvm-tools \
 	qemu-img \
 	make \
 	ansible \
 	libvirt \
 	libvirt-devel
 
+# rsync is not installed by default on CentOS 8
+[ ${CENTOS_VERSION} = "8" ] && dnf -y install rsync || true
```

Comment:
why the `|| true`? If installing `rsync` fails, will the job not eventually fail either?

Reply:
I added `|| true` because if the condition results in _false_ i.e, `CENTOS_VERSION != 8` return value will be 1 which is not what we want as the outcome. Script still needs to continue. But as you mentioned if somehow _rsync_ installation fails, we ignore it and continue as per the above change(but will fail at some other point). What do you suggest here? A clean **if** condition?

Reply:
ah, right, that seems to be a difference from the usual, more verbose
```shell
if [ ${CENTOS_VERSION} = "8" ]
then
    dnf -y install rsync
fi
```
notation

Maybe phrase the comparing and do it like this:
```shell
# only need to install rsync on CentOS 8 and newer
[ ${CENTOS_VERSION} -le "8" ] || dnf -y install rsync
```

Reply:
> `[ ${CENTOS_VERSION} -le "8" ] || dnf -y install rsync`

I hope you meant `lt` and not `le`